README.md

```diff
@@ -39,7 +39,28 @@ Please file an issue if additional packages are required.
 * python3.9
 * python3.9-venv
 * musl-1.2.2
-* ARM GCC compiler; 10.2-2020.11
+* cmake
+* ninja-build
+* ARM GCC compiler for none-elf; version 12.2.1 20221205
+* device tree compiler
+* xmllint
+
+To build the documentation you also need
+* pandoc
+* pdflatex
+* texlive-latex-recommended
+* texlive-fonts-recommended
+* texlive-fonts-extra
+* texlive-latex-extra
+
+On a Debian-like system you can do:
+
+    $ sudo apt install build-essential git cmake ninja-build \
+	    device-tree-compiler libxml2-utils \
+		pandoc texlive-latex-base texlive-latex-recommended \
+		texlive-fonts-recommended texlive-fonts-extra \
+		python3 python3-venv \
+		musl-dev musl-tools
```

Comment:
There's been multiple issues when using the apt packaged musl, I'm wondering whether we should recommend building from source here. We can keep the apt install for now maybe and see how many people run into issues.

`musl-dev` is not needed.

Reply:
musl-dev includes the actual library; musl-tools has musl-gcc that uses it.  musl-dev will be pulled in when musl-dev is installed; this make the dependency explicit.

Reply:
Yes you are correct, my bad. I'm fine with having it explicit.